### ISSUE TITLE
[Bug] showPasswordVisibilityToggler option doesn't seem to work

### ISSUE THREAD

**Issue by altayevrim** [Laravel-Backpack/theme-tabler#176]
# Bug report
When I update the `showPasswordVisibilityToggler` option on `config/backpack/theme-tabler.php` file nothing happens.
### What I did

I updated `config/backpack/theme-tabler.php` file and set the `showPasswordVisibilityToggler` to `true`
### What I expected to happen

A show password toggle element should be visible.

### What happened

Nothing happened.

### What I've already tried to fix it

Checked and searched for `showPasswordVisibilityToggler` but it didn't appear on my project any other place other than `config/backpack/theme-tabler.php` file.  
I checked on Safari `Version 17.4.1 (19618.1.15.11.14)` and Google Chrome `Version 124.0.6367.62 (Official Build) (arm64)`

### Context to replicate the bug - Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

```
### PHP VERSION:
8.3.6

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, FFI, fileinfo, filter, ftp, gd, gmp, json, iconv, SPL, session, intl, standard, ldap, mbstring, mysqlnd, mysqli, pcntl, exif, imap, PDO, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, random, readline, redis, Reflection, imagick, shmop, SimpleXML, soap, sockets, sodium, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, Zend OPcache

### LARAVEL VERSION:
11.3.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/activity-log: 2.0.3
backpack/basset: 1.3.2
backpack/crud: 6.7.7
backpack/generators: v4.0.5
backpack/logmanager: v5.0.1
backpack/pro: 2.1.12
backpack/settings: 3.1.1
backpack/theme-tabler: 1.2.8
```

**Comment by pxpm** [comment]
Hey @altayevrim 

It's highly possible that you have the `form.blade.php` file in your `resources/views/vendor/backpack/theme-tabler/auth/login/inc` folder, so you are not getting the updates we do on the themes files. 

Here is where it lives in theme files: https://github.com/Laravel-Backpack/theme-tabler/blob/98bb65a5064930e372d7115d3463ffc44eb94cc0/resources/views/auth/login/inc/form.blade.php#L32 and there is also some Javascript needed at the top of that file.

I've just tested it and seems to be working as expected: 
![image](https://github.com/Laravel-Backpack/theme-tabler/assets/7188159/283e3bcc-8fa2-4788-a79b-e3725772d588)

I will be closing this as I don't think there is any issue here form our side, if I am wrong, and I am a lot of times, please re-open or ping me. 

Cheers

**Comment by altayevrim** [comment]
Hi, I fixed it. Somehow even though I created the project freshly 2-3 days ago. I needed to run composer update to get the update again. Also there is only `menu_items.blade.php` on the `resources/views/vendor/backpack/*` directory.

I thought the ** options.showPasswordVisibilityToggler** option would change all password fields, including the CRUD and profile. It seems it only changes the login form.
